CONTRIBUTING.md

```diff
@@ -124,15 +136,23 @@ action (e.g. `.collect()`).
 ### Submitting PRs
 
 When submitting a PR, make sure to use the preformatted template.
+Except in special cases, all PRs should be against `master`.
+Avoid using "staging branches" as much as possible.
+If you want to add complicated features, please
+[stack your PRs](https://graysonkoonce.com/stacked-pull-requests-keeping-github-diffs-small/)
```

Comment:
This seems good in principle, but very complicated. Have you actually seen anyone do this, (as opposed to just breaking into multiple PRs and sending them for review one after the other)?

Reply:
It’s not so complicated! We did it quite a bit for core Snorkel development on v0.9!